packages/playwright-core/src/server/chromium/crBrowser.ts

```diff
@@ -485,6 +470,8 @@ export class CRBrowserContext extends BrowserContext {
   async doUpdateRequestInterception(): Promise<void> {
     for (const page of this.pages())
       await (page._delegate as CRPage).updateRequestInterception();
+    for (const sw of this.serviceWorkers())
```

Comment:
I think we should also cover `setExtraHTTPHeaders`, `setOffline` and `doSetHTTPCredentials`.